### PR TITLE
feat: add globals, format #43 #44

### PR DESCRIPTION
**Pull request by daybrush**
#43 #44

### CODE REVIEW

File: test/.storybook/manager.js [hunk 1]
```diff
@@ -1,5 +1,32 @@
 import { addons } from "@storybook/addons";
+import * as prettier from "prettier/standalone";
+import * as htmlParser from "prettier/parser-html";
+import * as babelParser from "prettier/parser-babel";
+import * as postCSSParser from "prettier/parser-postcss";
+
 
 addons.setConfig({
-    panelPosition: "right",
+    panelPostion: "right",
```
Comment (WoodNeck) [inline test/.storybook/manager.js:9]: Missing "i" here I guess? `panelPostion` -> `panelPosition`

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/PreviewPanel.tsx [hunk 2]
```diff
@@ -1,27 +1,24 @@
 import * as React from "react";
-import { useArgs, useParameter, useChannel } from "@storybook/api";
+import { useArgs, useParameter, useChannel, useGlobals } from "@storybook/api";
 import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
-import * as Prism from "prismjs";
-
-import "prismjs/components/prism-typescript";
```
Comment (ryuran) [inline src/PreviewPanel.tsx:6]: Do we still have a direct dependency to prism ?
If no we can remove prism from the package.json
https://github.com/naver/storybook-addon-preview/blob/master/package.json#L33-L40

Comment (daybrush) [reply src/PreviewPanel.tsx:6]: @ryuran 

Removed prism.
Test storybook-addon-preview@2.3.0-beta.1

**Comment by ryuran** [comment]
Tested `storybook-addon-preview@2.3.0-beta.1`
I set this formater:
```js
 previewFormatter: (format, code) => {
    if (format) {
      return prettier.format(removeEmptyLines(code), {
        parser: "html",
        plugins: [
          htmlParser,
        ],
      });
    }
    return code;
  },
```
where removeEmptyLine is :
```js
removeEmptyLines = (template) =>  template.trim().replace(/(\r\n|\n|\r)/gm, '\n').replace(/\n\s*\n/gm, '\n');
```

**Comment by daybrush** [comment]
@ryuran 

Are there any problems using it?

I think I will release it like this.

**Comment by ryuran** [comment]
Non problem using it.
I tested globals with knobs and with args.
It works as expected

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**